clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10513,7 +10513,8 @@ static void getTripleBasedSPIRVTransOpts(Compilation &C,
               ",+SPV_INTEL_tensor_float32_conversion"
               ",+SPV_INTEL_optnone"
               ",+SPV_KHR_non_semantic_info"
-              ",+SPV_KHR_cooperative_matrix";
+              ",+SPV_KHR_cooperative_matrix"
+              ",+SPV_EXT_shader_atomic_float16_add";
```

Comment:
This might be because I have not followed this work closely. But I curious why we have '_add' here. Is float6 not supported for other ops?

Thanks

Reply:
> Is float6 not supported for other ops?

Yes, we have [SPV_EXT_shader_atomic_float_add](http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/EXT/SPV_EXT_shader_atomic_float_add.html) that adds OpAtomicFAddEXT. And [SPV_EXT_shader_atomic_float16_add](http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/EXT/SPV_EXT_shader_atomic_float16_add.html) that allows the instruction to work with 16bit float.